src/components/page-layout/page-layout.js

```diff
@@ -15,43 +14,19 @@ import layoutConfig from './layout.config.js';
 import SiteSearchAPIConnector from '@elastic/search-ui-site-search-connector';
 
 export default class PageLayout extends React.Component {
-  // render the page's top bar navigation
-  renderTopbar = (switchedNavigation, parentPath) => {
-    const {
-      constants,
-      topBarSticker,
-      tabListAppend,
-      hideTopBar,
-      hideSearch
-    } = this.props;
-    if (hideTopBar) return null;
-    return (
-      <PageLayoutTopbar
-        {...this.props}
-        constants={constants}
-        navigation={switchedNavigation}
-        parentPath={parentPath}
-        topBarSticker={topBarSticker}
-        tabListAppend={tabListAppend}
-        hideSearch={hideSearch}
-      />
-    );
-  };
-
   // render the page's sidebar
   renderSidebar = (config, switchedNavigation, parentPath) => {
-    const { customSidebar, headings, topBarSticker } = this.props;
+    const { customSidebar, constants } = this.props;
     return (
       config.sidebar !== 'none' && (
-        <div className={`col col--4-mm col--12 ${config.sidebarTheme}`}>
+        <div className={`col col--3-mm col--12 ${config.sidebarTheme}`}>
```

Comment:
I decreased the columns for the sidebar now that it's a bit simpler and added a column to the content section below.